pkg/common/providers/crc/crc.go

```diff
@@ -285,3 +285,8 @@ func getPullSecretFileContent() (string, error) {
 	}
 	return crcPullSecret, nil
 }
+
+// ExtendExpiry extends the expiration time of an existing cluster
+func (m *Provider) ExtendExpiry(clusterID string, hours uint64, minutes uint64, seconds uint64) error {
+	return nil
```

Comment:
Let's return an error here, fmt.Errorf("ExtendExpiry is unsupported by CRC clusters.")